spring/boot-webflux-autoconfigure/src/main/java/com/linecorp/armeria/spring/web/reactive/ArmeriaReactiveWebServerFactory.java

```diff
@@ -100,6 +94,38 @@ public ArmeriaReactiveWebServerFactory(ConfigurableListableBeanFactory beanFacto
         this.beanFactory = requireNonNull(beanFactory, "beanFactory");
     }
 
+    private com.linecorp.armeria.spring.Ssl toArmeriaSslConfiguration(Ssl ssl) {
+        ClientAuth clientAuth = null;
+        if (ssl.getClientAuth() != null) {
+            switch (ssl.getClientAuth()) {
+                case NEED:
+                    clientAuth = ClientAuth.REQUIRE;
+                    break;
+                case WANT:
+                    clientAuth = ClientAuth.OPTIONAL;
+                    break;
+            }
+        }
+        if (!ssl.isEnabled()) {
+            return new com.linecorp.armeria.spring.Ssl();
+        }
+        return new com.linecorp.armeria.spring.Ssl()
+                .setEnabled(ssl.isEnabled())
+                .setClientAuth(clientAuth)
+                .setCiphers(ImmutableList.copyOf(firstNonNull(ssl.getCiphers(), EMPTY)))
```

Comment:
Don't we pass `null` if `ssl.getCiphers()` returns `null`? Because https://github.com/line/armeria/pull/1663/files#diff-ea7de1d469d2e28e1dff38bcc68680c6R373 sets if it's not `null`.
It's the same about the `setEnabledProtocols`.